netutils/bandwidth.py

```diff
@@ -0,0 +1,527 @@
+
+    Returns:
+        float: Speed in Tbps converted from Gbps.
+    """
+    return speed / 1000
+
+
+def gbps_to_tbytes(speed: float) -> float:
+    """Method to convert `speed` from Gbps to TBps value.
+
+    Args:
+        speed (float): Speed in Gbps to convert to TBps.
+
+    Returns:
+        float: Speed in TBps converted from Gbps.
+    """
+    return speed / 8000
+
+
+def tbps_to_kbps(speed: float) -> float:
+    """Method to convert `speed` from Tbps to Kbps value.
+
+    Args:
+        speed (float): Speed in Tbps to convert to Kbps.
+
+    Returns:
+        float: Speed in Kbps converted from Tbps.
+    """
+    return speed * 1000000000
+
+
+def tbps_to_kbytes(speed: float) -> float:
+    """Method to convert `speed` from Tbps to KBps value.
+
+    Args:
+        speed (float): Speed in Tbps to convert to KBps.
+
+    Returns:
+        float: Speed in KBps converted from Tbps.
+    """
+    return speed * 125000000
+
+
+def tbps_to_mbps(speed: float) -> float:
+    """Method to convert `speed` from Tbps to Mbps value.
+
+    Args:
+        speed (float): Speed in Tbps to convert to Mbps.
+
+    Returns:
+        float: Speed in Mbps converted from Tbps.
+    """
+    return speed * 1000000
+
+
+def tbps_to_mbytes(speed: float) -> float:
+    """Method to convert `speed` from Tbps to MBps value.
+
+    Args:
+        speed (float): Speed in Tbps to convert to MBps.
+
+    Returns:
+        float: Speed in MBps converted from Tbps.
+    """
+    return speed * 125000
+
+
+def tbps_to_gbps(speed: float) -> float:
+    """Method to convert `speed` from Tbps to Gbps value.
+
+    Args:
+        speed (float): Speed in Tbps to convert to Gbps.
+
+    Returns:
+        float: Speed in Gbps converted from Tbps.
+    """
+    return speed * 1000
+
+
+def tbps_to_gbytes(speed: float) -> float:
+    """Method to convert `speed` from Tbps to GBps value.
+
+    Args:
+        speed (float): Speed in Tbps to convert to GBps.
+
+    Returns:
+        float: Speed in GBps converted from Tbps.
+    """
+    return speed * 125
+
+
+def name_to_kbits(speed: str) -> int:
+    """Method to convert a short bandwidth name to int value in Kbps.
+
+    Args:
+        speed (str): Bandwidth to be converted like `100Gbps` to Kbps.
+
+    Returns:
+        int: int value of bandwidth to be converted to Kbps
+    """
+    if re.search("[mM]bps", speed):
+        _value = int(mbps_to_kbps(int(re.sub("[mM]bps", "", speed))))
+    if re.search("[gG]bps", speed):
+        _value = int(gbps_to_kbps(int(re.sub("[gG]bps", "", speed))))
+    return _value
+
+
+def name_to_bits(speed: str) -> int:
+    """Method to convert a short bandwidth name to int value in bps.
+
+    Args:
+        speed (str): Bandwidth to be converted like `100Gbps` to bps.
+
+    Returns:
+        int: int value of bandwidth to be converted to bps
+    """
+    if re.search("[mM]bps", speed):
+        _value = name_to_kbits(speed) * 1000
```

Comment:
Looks like this like is the same as below, we should be able to combine them into a single if statement

Reply:
That's actually wrong. I have the wrong multiplier for Gbps I believe. I've corrected this.